compiler/damlc/daml-ide-core/src/Development/IDE/Core/Rules/Daml.hs

```diff
@@ -432,14 +437,17 @@ ofInterestRule = do
                 mbVrs <- use RunScenarios file
                 forM_ (fromMaybe [] mbVrs) $ \(vr, res) -> do
                     let doc = formatScenarioResult world res
-                    when (vr `Set.member` openVRs) $
+                    when  (vr `Set.member` openVRs) $
                         sendEvent $ vrChangedNotification vr doc
-        -- We don’t always have a scenario service (e.g., damlc compile)
-        -- so only run scenarios if we have one.
+
+        -- We don’t always have a scenario service (e.g., damlc
```

Comment:
Not sure splitting up the command name over two lines makes it easier to read.

Reply:
You're referring to lines 445 - 450 here? You don't like it?

Reply:
No just changing the comment in line `437-438`. I found the previous layout easier to read (and apart from that, changing it in this PR is just noise).

Reply:
Oh ok. It's the result of emacs `M-q` (fill-paragraph). I'll restore it.